### DISCUSSION TITLE
RavenDB Cloud in AWS Switzerland

### DISCUSSION THREAD

**Discussion by bobskigit** [ravendb/ravendb#15254] [Q&A] [answered]
When can we expect ravenDB cloud to be available in AWS Switzerland?

**Comment by gregolsky** [comment]
Hello,

We are going to work on that soon - need to make sure it has all the required bells and whistles. I think we can expect it in end of November or early December.

**Comment by gregolsky** [comment, marked as answer]
AWS Switzerland is now available in RavenDB Cloud.
